src/app/feeds/feeds.html

```diff
@@ -175,7 +175,6 @@ <h5 class="bold">Who to follow</h5>
             <hr />
             <a
               ui-sref="vicigo.profile({ profileId: user.username })"
-              target="_self"
```

Comment:
why is it removed?
This does not fix the problem of v1->v2 profile forwarding. If you refresh the page, you will land anyway.
The problem here is not with the client side.

Reply:
https://github.com/honest-cash/honestcash/commit/0dede3057336178dce71891598acf1252ca25bc3#diff-5054e1edd843ce83e2e63e98a202e302R100

I told you over Slack that I had accidentally committed it thus creating the bug, then removal was not commited. this is the fix that reverts it to original non buggy state

Reply:
This still does not fix the problem with the forwarding. It's not a problem with the client but with the server.

Reply:
For example:
https://beta.honest.cash/profile/honest_cash will always forward you to V2.

Reply:
![2019-07-05_18-38-21](https://user-images.githubusercontent.com/15078042/60732951-2054ac00-9f54-11e9-9689-b8a8bc8b7d1d.gif)
can you atleast run it on local if not deploy to test because it works...